build.sbt

```diff
@@ -94,24 +82,24 @@ lazy val buildSettings = Seq(
   Compile / doc / sources := { if (isDotty.value) Seq() else (Compile / doc / sources).value }
 )
 
-lazy val catsVersion  = "2.3.0"
-lazy val dottyVersions = Seq("3.0.0-M1", "3.0.0-M2")
+lazy val catsVersion  = "2.3.1"
+lazy val dottyVersions = Seq("3.0.0-M3")
```

Comment:
Any reason not to keep both `M2` and `M3` it has been suggested to keep the latest two versions to simplify releasing.

Reply:
I tried but it seems that refined doesn't publish M2 for the latest version.

Reply:
Ok, the two versions idea should be used only when it doesn't create difficulties